airflow/providers/amazon/aws/transfers/s3_to_ftp.py

```diff
@@ -27,12 +27,10 @@
 class S3ToFTPOperator(BaseOperator):
     """
     This operator enables the transferring of files from S3 to a FTP server.
```

Comment:
```suggestion
    This operator enables the transferring of files from S3 to a FTP server.

```
It's needed to build docs.